main.go

```diff
@@ -133,7 +142,7 @@ func main() {
 	for sig := range sigChan {
 		switch sig {
 		case syscall.SIGHUP:
-			if err := loadConfiguration(); err != nil {
+			if _, err := loadConfiguration(); err != nil {
```

Comment:
After re-loading the configuration the modules also need to re-configured in order to apply updated configuration.